packages/css/src/plugins/increase-specificity.ts

```diff
@@ -0,0 +1,31 @@
+import type { Plugin } from 'postcss';
+
+const INCREASE_SPECIFICITY_SELECTOR = ':not(#\\9)';
+
+/**
+ * Increase the specificity of classes generated in Compiled by prepending ":not(#\\9)".
+ * This rule should run after CSS declarations have been atomicized and should not affect
+ * the original generated class name.
+ *
+ * This means generated class names with / without the increased specificity are the same,
+ * so when running Compiled together with (in product) + without it (in platform) they will
+ * keep the deterministic behaviour of the last declared style wins, but enjoying increased
+ * specificity when used in non-Compiled contexts for migration purposes, e.g. xcss prop
+ * being passed to Emotion CSS-in-JS!
+ */
+export const increaseSpecificity = (): Plugin => {
+  return {
+    postcssPlugin: 'increase-specificity',
+    OnceExit(root) {
```

Comment:
Because we want this to run after the atomic plugin we need to use the same hook - OnceExit!

Reply:
TIL OnceExit, will be handy next time I do some babel plugin stuff :)

Reply:
This is postcss ;-)